pages/video-detail/video-detail.view.scss

```diff
@@ -150,8 +154,9 @@ strong {
 		ng-include {
 			fill: $text;
 		}
-		img, svg {
+		img, svg, i {
```

Comment:
If there are no more <svg> and we have replaced for <i>, we don't need svg tag in css anymore